app/helpers/fae/nested_form_helper.rb

```diff
@@ -2,9 +2,17 @@ module Fae
   module NestedFormHelper
 
     def th_columns(attribute)
-      attribute = (attribute.is_a?(Hash) && attribute[:title]) ? attribute[:title] : attribute
+      if (attribute.is_a?(Hash) && attribute[:title])
+        attribute = attribute[:title]
+      else
+        if ['updated_at', 'modified_at'].include? attribute.to_s
```

Comment:
This control could be simplified to `attribute = :modified if ['updated_at', 'modified_at'].include? attribute.to_s` since you're not returning the `attribute` from this method.